app/code/Shiphawk/Order/Observer/CheckConfiguration.php

```diff
@@ -44,7 +44,7 @@ public function execute(\Magento\Framework\Event\Observer $observer)
             $this->messageManager->addWarning('Missing information required for printing labels: Please add a store name and phone number under Configuration > General > Store Information');
         }
 
-        if (property_exists($response, 'error')) {
+        if (is_null($response) || property_exists($response, 'error')) {
```

Comment:
This change was made to prevent System errors when we failed to make request to check the Api Key. For example, when we specified invalid Custom Gateway Url.